vkcs/resource_vkcs_db_config_group.go

```diff
@@ -0,0 +1,215 @@
+package vkcs
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceDatabaseConfigGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDatabaseConfigGroupCreate,
+		ReadContext:   resourceDatabaseConfigGroupRead,
+		UpdateContext: resourceDatabaseConfigGroupUpdate,
+		DeleteContext: resourceDatabaseConfigGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
```

Comment:
don't we have a way to validate schema (parameters) before call of create/update?